docs/user/registering-microshift-devices-acm/registering-microshift-devices-acm.md

```diff
@@ -0,0 +1,184 @@
+# Auto-Registering Devices with MicroShift into ACM
+
+If you have fleets of devices running an OS image that includes MicroShift, you can configure these fleets to auto-register MicroShift clusters with Red Hat Advanced Cluster Management (ACM).
+
+Auto-registration relies on ACM's [agent registration](https://docs.redhat.com/en/documentation/red_hat_advanced_cluster_management_for_kubernetes/2.12/html/clusters/cluster_mce_overview#importing-managed-agent) method for importing clusters. That method allows fetching the Kubernetes resource manifests to install ACM's klusterlet agent and registering the agent through calls to a REST API. This REST API can be set up as configuration source for devices by creating a Repository resource and referencing that resource from the fleet's device template.
+
+This document is intended for ACM and FlightCtl developers, QE engineers, and users who want to explore this feature. It provides step-by-step instructions, assuming minimal prior knowledge of ACM or FlightCtl.
+
+## 1 Enable `edge-manager` in ACM
+
+### 1.1 Patch `edge-manager-preview` in MCH.
+
+```console
+oc patch mch multiclusterhub -n open-cluster-management \
+       --type=json -p='[{"op": "add", "path": "/spec/overrides/components/-","value":{"name":"edge-manager-preview","enabled":true}}]'
+```
+
+[!NOTE] If you're testing with a [downstream ACM](https://quay.io/repository/acm-d/acm-custom-registry?tab=tags), you also need to add the following annotation to the MCH, otherwise agents will not be able to use downstream images.
```

Comment:
I prefer to use the customized image registry here like if you have a private image registry and you are want to get images from your private image registry, you can config the imageRepository like below